ui/app/common/directives/views/pnc-select-items.html

```diff
@@ -0,0 +1,41 @@
+<!--
+
+    JBoss, Home of Professional Open Source.
+    Copyright 2014 Red Hat, Inc., and individual contributors
+    as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<ul ng-class="{'list-group': singleItem === 'false' , 'list-group-no-bottom-margin': singleItem === 'true' }" ng-show="shouldShowList()">
+    <li class="list-group-item" ng-repeat="item in selectedItems">
+        {{ item.fullDisplayText }}
+        <button type="button" class="close" aria-label="Close" ng-click="removeItem(item)">
+            <span aria-hidden="true">×</span>
+        </button>
+    </li>
+</ul>
+<div class="pnc-infinite-select" ng-show="shouldShowSelection()">
+
+    <input ng-required="{{ selectRequired }}" id="{{ selectId }}" name="{{ selectId }}" 
```

Comment:
Hi @vibe13 ,  `name="{{ selectId }}"` - there is I think small typo mistake, `name="{{ selectName }}` should be probably used?